lib/theme_check/theme.rb

```diff
@@ -40,6 +43,11 @@ def default_locale_json
       end
     end
 
+    def create_default_locale_json
+      @default_locale_json = JsonFile.new('locales/en.default.json', @storage)
+      @default_locale_json.touch
```

Comment:
Should it be empty or should it at least contain an empty object? Would our platform accept a file that is empty?

```json
{}
```

Reply:
Yes, it needs to contain `{}`